kDrive/UI/Controller/Menu/MenuViewController.swift

```diff
@@ -281,21 +283,23 @@ extension MenuViewController {
         case .disconnect:
             let alert = AlertTextViewController(title: KDriveResourcesStrings.Localizable.alertRemoveUserTitle,
                                                 message: KDriveResourcesStrings.Localizable
-                                                    .alertRemoveUserDescription(currentAccount.user.displayName),
+                                                    .alertRemoveUserDescription(currentAccount?.user.displayName ?? ""),
                                                 action: KDriveResourcesStrings.Localizable.buttonConfirm,
                                                 destructive: true) {
-                self.accountManager.removeTokenAndAccount(account: self.accountManager.currentAccount)
+                if let currentAccount = self.accountManager.currentAccount {
```

Comment:
This PR is just change over existing code, but this `didSelectRowAt` method does a lot.
The log-out actions should probably be handled by a dedicated service.